lib/src/modules/keyboard_module/ios/ios_english_keyboard.dart

```diff
@@ -10,25 +10,20 @@ class IOSEnglishKeyboard extends StatelessWidget
   const IOSEnglishKeyboard({super.key});
 
   void _onCharacterTap(String character) {
-    final eventData = RawKeyEventDataIos(
-      characters: character,
-      charactersIgnoringModifiers: character,
-      keyCode: 0,
-      modifiers: 0,
-    );
-
     SystemTextInputChannelInterceptor.instance.activeIME.handleKeyEvent(
-      RawKeyDownEvent(
-        data: eventData,
+      KeyDownEvent(
+        physicalKey: _findPhysicalKey(character),
+        logicalKey: _findLogicalKey(character),
```

Comment:
I couldn't find a better way for this than to make a long switch statement for each character. Can you think of anything better?